### DISCUSSION TITLE
P1P2 Pinout for EHSXB08P30E

### DISCUSSION THREAD

**Discussion by nicx** [Arnold-n/P1P2MQTT#72] [General]
Hey, I think I have "tomatoes on my eyes"... maybe someone can help where to find the correct P1P2 pins on my EHSXB08P30E?

**Comment by KalaNagHTD** [comment]
If it is the old Version of the ECH2O with just one central knob (instead of two) the P1P2 functionality does not exist.Am 18.07.2023 um 08:48 schrieb nicx ***@***.***>:﻿
Hey, I think I have "tomatoes on my eyes"... maybe someone can help where to find the correct P1P2 pins on my EHSXB08P30E?

**Comment by nicx** [comment]
@KalaNagHTD it is the new one with 2 knobs :)

**Comment by KalaNagHTD** [comment]
Look for connector X15M on the mainboard (pins 11 and 12)Am 18.07.2023 um 09:28 schrieb nicx ***@***.***>:﻿
@KalaNagHTD it is the new one with 2 knobs :)

**Comment by nicx** [comment]
@KalaNagHTD found it, its labeled "User Interface". So I assume P1 is pin 11 and P2 is pin 12?

**Comment by KalaNagHTD** [comment]
According to the manual 11-12 = P1-P2Am 18.07.2023 um 11:07 schrieb nicx ***@***.***>:﻿
@KalaNagHTD found it, its labeled "User Interface". So I assume P1 is pin 11 and P2 is pin 12?

**Comment by Arnold-n** [comment]
Indeed, but P1 and P2 can be  swapped; although there is a polarity, all controllers accept wiring either way.
